modules/programs/rofi.nix

```diff
@@ -405,7 +414,13 @@ in {
       inherit value;
     };
 
-    home.packages = [ cfg.package ];
+    home.packages = let
+      rofiWithPlugins = cfg.package.override
+        (old: rec { plugins = (old.plugins or [ ]) ++ cfg.plugins; });
+    in if builtins.hasAttr "override" cfg.package then
```

Comment:
this `if` statement feels like it should be in the `let` block?

Reply:
Do you mean something like this?
```nix
home.packages = let    
  rofiWithPlugins = if builtins.hasAttr "override" cfg.package then 
    cfg.package.override     
    (old: rec { plugins = (old.plugins or [ ]) ++ cfg.plugins; })
  else
    cfg.package;     
in [ rofiWithPlugins ];
```

Reply:
Sorry for the delay! That is what I was thinking. Or even something like:
```nix
home.packages = let
  rofiWithPlugins = cfg.package.override
    (old: rec { plugins = (old.plugins or [ ]) ++ cfg.plugins; });
  rofiPackage = if builtins.hasAttr "override" cfg.package then rofiWithPlugins else cfg.package;
in [ rofiWithPlugins ];
```